src/software/thunderscope/robot_diagnostics/robot_info.py

```diff
@@ -78,26 +89,36 @@ def __init__(self, robot_id, load_fullsystem, control_mode_signal):
 
         self.setLayout(self.layout)
 
-    def create_control_mode_menu(self, load_fullsystem):
+    def create_robot_status_expand_button(self):
+        """
+        Creates the button to expand / collapse the robot status view
+        :return: QPushButton object
+        """
+        button = QPushButton()
+        button.setCheckable(True)
+        button.setText("INFO")
+        return button
+
+    def create_control_mode_menu(
+        self, available_control_modes: List[IndividualRobotMode]
+    ):
         """
         Creates the drop down menu to select the input for each robot
         :param robot_id: the id of the robot this menu belongs to
-        :param load_fullsystem: whether fullsystem is also an option
+        :param available_control_modes: the currently available input modes for the robots
+                                        according to what mode thunderscope is run in
         :return: QComboBox object
         """
         control_mode_menu = QComboBox()
 
         control_mode_menu.addItems(
-            [
-                common_widgets.IndividualRobotMode.NONE.name,
-                common_widgets.IndividualRobotMode.MANUAL.name,
-            ]
+            [control_mode.name for control_mode in available_control_modes]
         )
-        control_mode_menu.setCurrentIndex(0)
 
-        if load_fullsystem:
-            control_mode_menu.addItem(common_widgets.IndividualRobotMode.AI.name)
+        if IndividualRobotMode.AI in available_control_modes:
             control_mode_menu.setCurrentIndex(2)
```

Comment:
oh that should be 2 sorry, before I had 2 different modes for XBox and Diagnostics so AI was 3, but since they're both sending the same primitives they're both under the Manual control mode now